MAUI/Barcode/two-dimensional-symbology.md

```diff
@@ -0,0 +1,273 @@
+                               ShowText="True"
+                               HeightRequest="300"
+                               WidthRequest="300">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:QRCode />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="5" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 300;
+    barcode.WidthRequest = 300;
+    barcode.Value = "www.syncfusion.com";
+    barcode.Symbology = new QRCode();
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator QR Code Symbology](images/two-dimensional-symbology/maui-qr-code-symbology.png)
+
+The data that can be stored in the QR code depends upon the following property:
+
+* `Error correction level`
+* `QR code version`
+* `Input mode`
+
+As like one dimensional symbology, the two dimensional symbology also supports the `Module` property.
+The property is used to define the size of the smallest line or dot of the barcode. If this property is not set, the size of the smallest dot of the barcode is calculated based on the available size.
+
+### Error correction level
+
+The `ErrorCorrectionLevel` property employs error correction to generate a series of error correction code words that are added to the data code word sequence in order to enable the symbol to withstand damage without loss of data. By default, its value is set as `Auto`.
+
+Low - it recovers the data up to 7%
+Medium - it recovers the data up to 15%
+Quartile - it recovers the data up to 25%
+High - it recovers the data up to 30%
+Auto - it calculates error correction level automatically based on `InputMode` and `CodeVersion`.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="6" %}
+
+    <barcode:SfBarcodeGenerator Value="www.syncfusion.com"
+                                ShowText="True"
+                                HeightRequest="300" 
+                                WidthRequest="300">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:QRCode ErrorCorrectionLevel="High"/>          
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="7" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 300;
+    barcode.WidthRequest = 300;
+    barcode.Value = "www.syncfusion.com";
+    barcode.Symbology = new QRCode() 
+    { 
+        ErrorCorrectionLevel = ErrorCorrectionLevel.High 
+    };
+
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator QR Code error correction level high](images/two-dimensional-symbology/maui-qr-code-error-correction-level-high.png)
+
+The data can be read from the damaged image based on the error correction level,
```

Comment:
The data can be read from the damaged image based on the error correction level.

Reply:
Changes updated.